iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/port_introspection.inl

```diff
@@ -195,38 +204,39 @@ bool PortIntrospection<SenderPort, ReceiverPort>::PortData::updateConnectionStat
     return true;
 }
 
-template <typename SenderPort, typename ReceiverPort>
-bool PortIntrospection<SenderPort, ReceiverPort>::PortData::addSender(typename SenderPort::MemberType_t* port,
-                                                                      const ProcessName_t& name,
-                                                                      const capro::ServiceDescription& service,
-                                                                      const RunnableName_t& runnable)
+template <typename PublisherPort, typename SubscriberPort>
+bool PortIntrospection<PublisherPort, SubscriberPort>::PortData::addPublisher(
+    typename PublisherPort::MemberType_t* port,
+    const ProcessName_t& name,
+    const capro::ServiceDescription& service,
+    const RunnableName_t& runnable)
 {
     std::lock_guard<std::mutex> lock(m_mutex);
 
-    auto iter = m_senderMap.find(service);
-    if (iter != m_senderMap.end())
+    auto iter = m_publisherMap.find(service);
+    if (iter != m_publisherMap.end())
```

Comment:
@elBoberido you did not see this!!!!!!! Where is the BracketMan when you need him.

Reply:
that's inventory code. I just complain about new code ;)

Reply:
Good catch. Done.